### ISSUE TITLE
Feature request: Maybe the pretrained models on 3dshapes can be realeased

### ISSUE THREAD

**Issue by ryx19th** [google-research/disentanglement_lib#27]
As I understand they were not realeased because the 3dshapes dataset itself wasn't at that time. But it has become public now. If the corresponding pretrained models on it can be realeased too it would be really great.

**Comment by noahcao** [comment]
Yes, I'm also looking forward to that, or at least the raw experiments logs as other datasets.
